manifest.yml

```diff
@@ -1,7 +1,7 @@
 ---
 applications:
 - name: funding-service-design-notification-dev
-  memory: 64M
+  memory: 256M
```

Comment:
Why are we bumping the memory in this PR? Can we revert this file if it is unrelated?

Reply:
Maybe a separate PR for this one to keep SOC?